crucible/from-file-json-manipulation-api/README.md

```diff
@@ -0,0 +1,24 @@
+This example has an `oslat.json` template that the `run.sh` script
+ingests and adds information to (or deletes information from) based on
+the variables defined at the top of the `run.sh` using the "API" that
+is defined.
+
+The "API" uses `jq` to perform "well known" operations against the
+JSON template file which results in the creation of the
+`oslat.json.run` file.  This file is then submitted to crucible using
```

Comment:
I'd keep json files with the .json "extension" so users can edit these files with their preferred editors.

Reply:
Yeah, I regretted doing this later on for this reason but I was too lazy to change it at the time.  I agree that we should maintain the `.json` extension going forward.